chainer/functions/connection/convolution_2d.py

```diff
@@ -235,22 +230,36 @@ def _forward_grouped_convolution(self, x, W, b):
 
         xp = cuda.get_array_module(x)
 
-        _x = x.reshape(N, G, iCg, iH, iW)
+        _x = x.reshape((N, G, iCg, iH, iW))
```

Comment:
Are these changes on `reshape` arguments essential?
If not, could you undo them?

Reply:
The former form`x.reshape(N, G, iCg, iH, iW)` is not allowed when the type of `x` is `mdarray`. Therefore we need this change here for now. Maybe we can improve the implementation of ideep4py, though.

Reply:
I see. Thank you for clarification!